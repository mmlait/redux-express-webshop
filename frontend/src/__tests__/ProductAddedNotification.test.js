import React from 'react';
import ProductAddedNotification from '../components/ProductAddedNotification';
import { create } from 'react-test-renderer'

describe('ProductAddedNotification', () => {
    it('should render ProductAddedNotification component', () => {
    let tree = create(<ProductAddedNotification />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})