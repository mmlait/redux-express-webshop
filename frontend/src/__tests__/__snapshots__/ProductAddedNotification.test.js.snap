// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductAddedNotification should render ProductAddedNotification component 1`] = `
<div
  className="sc-bdVaJa gtzOcq"
>
  <p>
    A new product was added!
  </p>
</div>
`;
